codeforamerica/vita-min
# Fyst 807 1099 r create base table and polymorphic tables

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-807
## Is PM acceptance required? (delete one)
- No - merge after code review approval
## What was done?
- Added a StateFile1099R table, and gave it polymorphic references to the existing StateFileXxIntake tables and to new StateFileXx1099RFollowup tables. 
- Implemented synchronizing of 1099Rs in the DF XML over to the table when importing from an API response or submitting the federal info form
- Removed the previously-added 1099R aggregate fields from StateFileAzIntake along with the associated controller/form/view
## How to test?
- Start an AZ intake, select an XML with at least one 1099R, select "i want to edit XML" & note that it's on the federal info form
